src/index.js

```diff
@@ -1,42 +1,45 @@
 /* eslint-disable import/extensions */
-/* eslint-disable import/no-extraneous-dependencies */
 import './style.css';
+import { AddItemsToList, Store, RemoveItemFromList } from './modules/index';
 
-const tasks = [
-  {
-    description: 'Commit to GitHub',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Solve a challenge on hackerrank',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: 'Read a novel',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'Watch a movie',
-    completed: false,
-    index: 3,
-  },
-];
+// ========== Event Listener to display items ===========
+document.addEventListener('DOMContentLoaded', AddItemsToList.addListItemsToInterface);
 
-const info = () => {
-  const taskContent = document.querySelector('.list-items');
-  for (let i = 0; i < tasks.length; i += 1) {
-    const element = document.createElement('li');
-    element.classList.add('list-info');
-    element.innerHTML = `
-    <input type="checkbox" name="${tasks.index}" id="${tasks.index}" title="Check">
-    <label for="${tasks.index}" contentEditable="true">${tasks[i].description}</label>
-    <i class="material-icons">more_vert</i>
-    `;
-    taskContent.appendChild(element);
+// ========== Event Listener to add item to list ==========
+const form = document.querySelector('.add-form');
+const listInput = document.getElementById('add-to-list');
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const index = Store.getItems().length ? Store.getItems()[Store.getItems().length - 1].index + 1
+    : 0;
+  const item = { description: listInput.value, completed: false, index };
+  AddItemsToList.addItemToList(item);
+  AddItemsToList.addListItemsToInterface();
+  form.reset();
+});
+
+// ======= Event to remove Item from List ========
+const listItems = document.querySelector('.list-items');
+listItems.addEventListener('click', (e) => {
+  if (e.target.classList.contains('fa-trash-can')) {
+    RemoveItemFromList.removeItemFromList(e.target);
+    RemoveItemFromList.changeIndex();
```

Comment:
- [ ] Kindly write the name of the function right as you wrote it here `changeIndex` and you created it in the `remove_from_list` file `changeTaskIndex`.

```javascript
RemoveItemFromList.changeTaskIndex();
```